platform/pulpcore/app/serializers/status.py

```diff
@@ -0,0 +1,57 @@
+from gettext import gettext as _
+from rest_framework import serializers
+
+from pulpcore.app.serializers.task import WorkerSerializer
+
+
+class VersionsSerializer(serializers.Serializer):
```

Comment:
What about if we change the structure and instead of using a dictionary/hash like:

```json
{"pulpcore": "3.0.0"}
```

We could do something like this?

```json
[ {"component": "pulpcore", "version": "3.0.0"} ]
```